yaml/of_builder.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: of-builder
+  namespace: openfaas
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: of-builder
+    spec:
+      containers:
+      - name: of-builder
+        image: openfaas/of-builder:0.3.0
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8080
+          protocol: TCP
+        securityContext:
+          privileged: true
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: of-builder
+  namespace: openfaas
+  labels:
+    app: of-builder
+spec:
+  type: NodePort
+  ports:
+    - port: 8080
+      protocol: TCP
+      targetPort: 8080
+      nodePort: 31121
```

Comment:
The nodePort can be removed, it'd only be needed for testing the of-builder manually. Maybe we can use a [ClusterIP](https://kubernetes.io/docs/concepts/services-networking/service/) instead?